ci-operator/jobs/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master-presubmits.yaml

```diff
@@ -1,5 +1,62 @@
 presubmits:
   openshift/cluster-logging-operator:
+  - agent: kubernetes
```

Comment:
Huh is this job intentionally added? You added a single clause to the ci-op config, called `e2e-operator`, but added two jobs, which is strange. The `pull-ci-openshift-cluster-logging-operator-master-e2e-operator` job below is correctly generated from ciop config. This one looks like it would be generated from basically identical clause, but having `as: e2e` instead. This looks like a mistakenly added job?

Reply:
updated presubmit based on newly generated job

Reply:
@petr-muller updated to address this conflict